pyserini/encode/_tct_colbert.py

```diff
@@ -38,12 +38,11 @@ def __init__(self, model_name: str, tokenizer_name=None, device='cuda:0'):
             self.model.to(self.device)
             self.tokenizer = BertTokenizerFast.from_pretrained(tokenizer_name or model_name)
 
-    def encode(self, texts, titles=None, fp16=False, **kwargs):
+    def encode(self, texts, titles=None, fp16=False,  max_length=512, **kwargs):
         if titles is not None:
             texts = [f'[CLS] [D] {title} {text}' for title, text in zip(titles, texts)]
         else:
             texts = ['[CLS] [D] ' + text for text in texts]
-        max_length = 512  # hardcode for now
```

Comment:
@lintool the issue is fixed, it's because originally it's using 512 for tct-colbert but now the default max-length from the command line argument is 256. Fixing the input solves the issue.